cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/AbstractSchedulerService.java

```diff
@@ -169,6 +172,8 @@ public void schedule(Id.Program programId, SchedulableProgramType programType, S
       schedule(programId, programType, ImmutableList.of(schedule));
     }
 
+    // TODO remove this annotation once the Schedule class becomes abstract
+    @SuppressWarnings("deprecation")
```

Comment:
Yes that is what I have done, but I also deprecated the ``getCronEntry`` method of ``Schedule``, and the annotation suppresses the warning for that.